README.md

```diff
@@ -36,4 +36,67 @@ The id is unique and changes with every new p2000 message.
 
 Extracting data can be done with a template like:
 
-`{{ state_attr('sensor.p2000_zwolle', 'melding') }}`
\ No newline at end of file
+`{{ state_attr('sensor.p2000_zwolle', 'melding') }}`
+
+----------------------------------------------------------------------------------
+### lovelace Dashboard
+
+You need markdown and logbook for the following settings
+
+More information on your dashboard.<br> tekst in Dutch.<br>
+![afbeelding](https://user-images.githubusercontent.com/62996429/144426649-70ec4732-14e1-4856-aa40-d8b81e99f3c3.png)
```

Comment:
The images will be missing if merged; Please add them to the assets folder and use them like
`(./assets/screenshot01.png)`